config/config.txt

```diff
@@ -241,6 +241,12 @@ TICKLAG 0.5
 ## Uncomment this to let players see their own notes (they can still be set by admins only)
 #SEE_OWN_NOTES
 
+### Comment these two out to prevent notes fading out over time for admins.
+## Notes older then this will start fading out.
+NOTE_FRESH_DAYS 91.31055
```

Comment:
such a strange number on these

Reply:
that is 3 avg months.

Reply:
I meant the `.31055` part(same for the next number)

Reply:
as he said, it's an average.
a month is often taken to be 4 weeks but this is not consistent, some months have 28 days, others have 31.

Reply:
1 year = 365.2422 days (to account for leap years)

1 month = 365.2422/12 = 30.43685 days

30.43685 * 3 = 91.31055